change/@microsoft-teams-js-be8ebe71-3afc-4cba-b70e-aa45d1cec1d3.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added getUserInfo to LiveShareHost",
```

Comment:
Please update comment to match the API being added (also please surround API names/interfaces with backticks '\`')